packages/devextreme/testing/testcafe/tests/scheduler/appointmentForm/showAppointmentPopup.ts

```diff
@@ -29,6 +29,30 @@ test('Invoke showAppointmentPopup method shouldn\'t raise error if value of curr
   height: 600,
 }));
 
+test('Appointment popup shouldn\'t raise error if appoitment is recursive', async (t) => {
+  const scheduler = new Scheduler('#container');
+  await t.doubleClick(scheduler.getAppointment('Meeting of Instructors').element);
+  await t.click(Scheduler.getEditRecurrenceDialog().series);
+
+  const consoleMessages = await t.getBrowserConsoleMessages();
+  await t.expect(consoleMessages.error.length).eql(0);
```

Comment:
It seems to me that the testcafe will indicate anyway if there is error in the console. Or is it impossible for the test to be without an assert?

Reply:
Unfortunately, if I remove this `expect`, the test will pass. Seems that this error is raised not with `throw ...`, but with `console.error`, which isn't caught by testcafe